### ISSUE TITLE
Divergent behavior of test loss

### ISSUE THREAD

**Issue by MatsValk** [pythonlessons/TensorFlow-2.x-YOLOv3#110]
Hi, Thanks for your awesome piece of code. I am training an object detection model on a custom dataset for 150 epochs. For some reason, the validation loss always has the same shape as in the picture, independent of yolo type, image size or learning rates. I have already tried to implement the SGD and AMSGrad optimizers, as Adam is known to diverge, but that didn't change anything. I have also tried to set focal loss to 0 instead of 2 and to add more data augmentation, but also didn't help. Any idea why this is caused and how it can be fixed? Thanks in advance. 

<img width="346" alt="Screenshot 2020-12-17 at 12 32 14" src="https://user-images.githubusercontent.com/49404749/102482871-2b29d080-4064-11eb-9485-3ced1e0a2207.png">
